apps/src/sites/studio/pages/scriptOverview.js

```diff
@@ -0,0 +1,27 @@
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PlcHeader from '@cdo/apps/code-studio/plc/header';
+import { renderCourseProgress } from '@cdo/apps/code-studio/progress';
+
+$(document).ready(() => initPage());
```

Comment:
any reason not to just pass in `initPage` directly instead of wrapping it in another function?